tests/modules/services/devilspie2/default.nix

```diff
@@ -0,0 +1,4 @@
+{
+  devilspie2-configuration = ./configuration.nix;
+  devilspie2-service = ./service.nix;
```

Comment:
I feel like these tests can be combined into one, but I think it would be fine to leave separate as well.

Reply:
I agree, these tests should be combined.